docs/examples.md

```diff
@@ -441,3 +441,75 @@ new BaseJobBuilder(
 }
 ```
 
+
+### Using GHE PR Watcher
```

Comment:
you'll want to update this to account for the fact that it now supports both GHE and gh.com